infra-templates/pipeline/0/rancher-compose.yml

```diff
@@ -3,6 +3,7 @@ catalog:
   name: Pipeline
   version: v0.1.0
   minimum_rancher_version: 1.6.13-rc2
+  maximum_rancher_version: 1.6.13-rc1
```

Comment:
let's have this match whatever the `min` version is. I didn't notice that it was set to rc2 before.